 active-rfcs/0001-issue-helper.md

```diff
@@ -0,0 +1,105 @@
+- Feature Name: Issue Helper
+- Start Date: 2020-02-20
+- Reference Issues:
+- Implementation PR:
+
+# Summary
+
+We'll use GitHub Issue [contact_links](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) to create issues, and in contact_links we'll put an external web page called issue-helper to help us create issues in a standard format.
+
+# Motivation
+
+In the past, TiDB's code repository used GitHub Issues to create bug reports, but because GitHub Issues use markdown to organize and write the report content. This form is simple but not structured, and not easy to use, so information transmission is inefficient.
+
+Also, as our bug reports require more and more information, the issue templates become more and more complex, making it difficult for users or contributors to fill out the templates.
+
+So we can use GitHub Issue's contact_links to develop a web page to help and guide people through the process of filling out a bug report. This will help us collect bug reports in a better and more standard way and make it easier for users and contributors to fill them out.
+
+# Detailed design
+
+## Introducing GitHub Issue contact_links
+
+GitHub Issues provides a way to configure external links. We can use the following configuration to add an external contact link.
+
+```yaml
+// .github/ISSUE_TEMPLATE/config.yml
+blank_issues_enabled: false
+contact_links:
+  - name: Create new issue
+    url: https://prow.tidb.io/tichi/new-issue
+    about: Please use the following link to create a new issue.
+```
+
+The effect of using this feature can be seen in the [vuejs repository](https://github.com/vuejs/vue/issues/new/choose).
+
+## External contact links combined with GitHub Issues
+
+Once we enable external links, we can direct the user or contributor to a web page that we've developed, where we'll use a standard form for the user or contributor to fill out the report, and then we'll use the [GitHub Issue creation API](https://docs.github.com/en/github/managing-your-work-on-github/about-automation-for-issues-and-pull-requests-with-query-parameters#supported-query-parameters) to resubmit the form to GitHub and create the Issue.
+
+The API looks like this: https://github.com/pingcap/tidb/issues/new?title=example&body=FormContent
+
+## Issue Helper
```

Comment:
can you show some issue helper demo page, to help contributors to establish direct experience? 😄   @hi-rustin

Reply:
There are examples of antd and vue below, I think it might not be appropriate to place our own demos as the current ones are very rough and work on the test-dev repository instead of the tidb repository.

Reply:
But also welcome to try：http://47.100.245.243:5000/